doc/tutorials/Multi-PC.md

```diff
@@ -213,4 +170,65 @@ rostopic list
 rostopic pub /test std_msgs/String "Hello"
 #--worker-container context--
 rostopic echo /test
-```
\ No newline at end of file
+```
+
+### Troubleshooting
```

Comment:
I have tried to use the swarm approach also for Misty, but I was not able to make it working (I don't know why), so I went for the host network approach. I think it is worth to try the swarm approach again instead.

Reply:
I did have it working in my testing a while back...could revisit soon. I don't think that update should be part of this PR though.